src/compas_rhino/objects/__init__.py

```diff
@@ -55,10 +54,19 @@
 from compas.datastructures import Network
 from compas.datastructures import VolMesh
 
+import warnings
+
+warnings.warn(
+    "The objects module in compas_rhino is deprecated. Use the equivalent functionality from compas_ui instead",
```

Comment:
`compas_ui`? That doesn't (yet?) exist, does it? Is it a typo?

Reply:
it is in the making :)